Tests/Swift Tests/BugsnagSwiftPublicAPITests.swift

```diff
@@ -46,7 +46,6 @@ class BugsnagSwiftPublicAPITests: XCTestCase {
     let onBreadcrumbBlock: BugsnagOnBreadcrumbBlock = { (breadcrumb) -> Bool in return false }
     
     func testBugsnagClass() throws {
-        Bugsnag.start(withApiKey: "")
```

Comment:
This had to go as it now throws an Objective C exception, which can't be caught here :(